packages/pwa/app/components/_app/index.jsx

```diff
@@ -168,6 +176,25 @@ const App = (props) => {
                             href={getAssetUrl('static/img/global/apple-touch-icon.png')}
                         />
                         <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                        {/* Urls for all localized versions of this page (including current page)
+                        For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                        {SUPPORTED_LOCALES.map((locale) => (
+                            <link
+                                rel="alternate"
+                                hrefLang={locale.toLowerCase()}
+                                href={`${appOrigin}${buildUrlLocale(locale)}`}
+                                key={locale}
+                            />
+                        ))}
+                        {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
+                        <link
+                            rel="alternate"
+                            hrefLang={defaultLocale.slice(0, 2)}
+                            href={`${appOrigin}${buildUrlLocale(defaultLocale)}`}
+                        />
```

Comment:
This is for catching all other English variants. It suggests to search engine crawlers to show these users the app's default `en-GB` locale.

Google says:

> You may have specific URLs for English speakers in Ireland (en-ie), Canada (en-ca), and Australia (en-au), but should also provide a generic English (en) page for searchers in, say, the US, UK, and all other English-speaking locations. It can be one of the specific pages, if you choose.